manifests/c/CocCoc/CocCoc/120.0.6099.234/CocCoc.CocCoc.locale.vi-VN.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: CocCoc.CocCoc
+PackageVersion: 120.0.6099.234
+PackageLocale: vi-VN
+Publisher: Coc Coc Co., Ltd.
+PackageName: Coc Coc
+License: None
+Copyright: Copyright 2018 Google LLC
+ShortDescription: Coc Coc Browser

```

Comment:
Can you provide a better description for what this application is?

Reply:
description? like this?: Cốc Cốc is the web browser and search engine for Vietnamese people.(https://coccoc.com/en/about-us)

Reply:
resolve this since no one response

Reply:
resolve this since no one response.